src/Persistence/AbstractQuery.php

```diff
@@ -192,13 +254,19 @@ protected function setMode($mode): self
         return $this;
     }
 
+    /**
+     * Add condition to the query scope (leaves model scope intact).
+     */
     public function where($fieldName, $operator = null, $value = null): self
     {
         $this->scope->addCondition(...func_get_args());
 
         return $this;
     }
 
+    /**
+     * Limit scope to only records with $id (leaves model scope intact).
```

Comment:
`leaves model scope intact` is property of Query, thus it should be not duplicated in the comments

then comment is unneeded :)